src/Maestro/maestro-angular/src/app/page/subscriptions-table/subscriptions-table.component.html

```diff
@@ -0,0 +1,26 @@
+<ul class="nav nav-tabs">
+  <ng-container *ngFor="let item of subscriptionsList | keyvalue">
+    <li class='nav-item' >
+      <a [ngClass]="{ active: item.key===currentTab, 'nav-link': true}" style="cursor: pointer;"
+        (click)="currentTab=item.key">{{item.key}}</a>
+    </li>
+  </ng-container>
+</ul>
+<table class="table table-sm table-hover mb-0" style="white-space: nowrap;">
+  <thead>
+    <th>Source Repository</th>
+    <th>Enabled</th>
+  </thead>
+  <tbody>
+    <ng-container *ngFor="let item of subscriptionsList[currentTab]">
+      <tr>
+        <td>
+          {{item.sourceRepository}}
```

Comment:
Rather than tabs for branches, it might be simpler if the "Source Repository" was merely "Source" and included something like "[repository]@[branch]".

Reply:
If you don't want tabs, how do you want it handled for cases where the same subscription is referenced from multiple branches? Multiple rows?

Reply:
That was my thinking.  Are there situations where that would be weird?

Reply:
My assumption was that the subs were edited per branch, which makes the tab grouping a more logical way to view them, but I will admit that I don't really know what would make the most sense for repo owners in this case.

Reply:
Something more like this?

![image](https://user-images.githubusercontent.com/44816719/66969051-d5f7c680-f03c-11e9-90f3-831d6e6e8f76.png)
![image](https://user-images.githubusercontent.com/44816719/66969089-faec3980-f03c-11e9-9d57-31e58d8be07f.png)

Reply:
Or maybe "Output Branch" instead of "Branch" to be extra clear?

Reply:
Under the first section, for "No dependencies".  I'd expect there to be branch information still in that scenario, and then the "no dependencies" info to be in the "Status" column.  Perhaps that's what you mean by "misaligned"?  If so, great, glad we're on the same page.

For the "Dependencies Without Subscriptions" section, do we have the dependency info?  ie, could we make the table look something like this?

| Dependency without a subscription | Source repository |
| --------------------------------------- | ------------------- |
| System.Foo.dll                                   | https://github.com/dotnet/corefx |
| System.Fun.dll                                   | https://github.com/dotnet/corefx |

It would probably help to run this by @mmitche and get his thoughts on what would be the most useful view here.

In general, this is getting close.  Really appreciate the screenshots.

Reply:
Yes on the first point - it just needed the branch column added.

I can do the Dependencies Without Subscriptions by individual dependency, but it'll be long (1-59 lines per missing subscription). I'm worried it's going to be too long, especially alongside the conflicting subscriptions work I did last night. If you think that information is going to be something that people want then we might want to have those two under optionally expanding sections that can be collapsed. (I'm saying this with no idea if that's possible or how to do it, just with the assumption that because I've seen it on other websites that it exists.)

![image](https://user-images.githubusercontent.com/44816719/67031252-f6b82e80-f0c5-11e9-8046-124f9fd7f0bb.png)

Reply:
![image](https://user-images.githubusercontent.com/44816719/67039936-494e1680-f0d7-11e9-9b07-6ba253d45cfa.png)

![image](https://user-images.githubusercontent.com/44816719/67039965-5834c900-f0d7-11e9-894b-8d545c2a7699.png)

![image](https://user-images.githubusercontent.com/44816719/67039999-6256c780-f0d7-11e9-8b0c-94642facfeb1.png)

Reply:
I don't think the "dependencies without subscriptions" section is needed.  

With regards to the dependencies, how do you know the branch?  I don't think you can since multiple branches could provide the dependency and it's up to the repo owner to figure out which one is needed.

"Show / Hide Dependencies" doesn't seem like the right label because it implies (to me), that I'm going to get information about all dependencies (not just dependencies missing subscriptions), but that's not what this table is about.  Also, it occurs to me that there is an overload of the term "dependencies".  In version.details.xml, dependencies references the assets desired.  In the db, dependencies refers to the repo, not the asset and the items are called assets.  So maybe "show / hide assets without subscriptions"?

What would it look like if a different repo started providing an asset?  ie, if a different repo starts producing an asset then you might have to decide to get an asset from dotnet/repoA@master or dotnet/repoB@release/3.1.

There's enough concrete visualization here to have a meaningful conversation with Mitchell.  Sync with him, summarize the conversation outcome here, and then let's get in a room and agree upon the desired result rather than forcing you to go back and forth on these iterations.  Sound reasonable?